Dockerfile

```diff
@@ -1,22 +1,19 @@
-# ubuntu 16.04 xenial image
-FROM ubuntu
+# use base python
+FROM python:2.7
 
-# apt update it
-RUN apt-get update
-# get essentials
-RUN apt-get install -y tar git curl wget dialog net-tools build-essential python python-dev python-distribute python-pip
```

Comment:
I can confirm that we don't need those. I was following a docker example and had it there for a while. All the stuff needed by the app is in the python requirements.txt file. Also since the tests passed, we're good in removing the apt-get installs.